README.md

```diff
@@ -474,3 +474,10 @@ an invite via [http://bit.ly/slackelixir](http://bit.ly/slackelixir).
 The Slack channel is appropriate for quick questions or general design
 discussions.  The Slack discussion is archived at
 [http://slack.elixirhq.com/kafkaex](http://slack.elixirhq.com/kafkaex).
+
+## default snappy algorithm use snappyer package
```

Comment:
it looks like the default is actually snappy - should we change this to say "Changing the default snappy library to snappyer"?

Reply:
@getong please consider @joshuawscott recommendation here thanks!